Examples/Algorithms/Digitization/src/DigitizationOptions.cpp

```diff
@@ -0,0 +1,234 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/Digitization/DigitizationOptions.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "Acts/Utilities/ParameterDefinitions.hpp"
+#include "ActsExamples/Digitization/Smearers.hpp"
+#include "ActsExamples/Utilities/Options.hpp"
+#include <boost/program_options.hpp>
+
+#include <string>
+
+void ActsExamples::Options::addDigitizationOptions(
+    ActsExamples::Options::Description& desc) {
+  using boost::program_options::value;
+  using namespace std;
+
+  auto opt = desc.add_options();
+  opt("digi-input-hits", value<string>()->default_value("hits"),
+      "Name of the input hit collection.");
+  opt("digi-config-file", value<string>()->default_value(""),
+      "Configuration (.json) file for digitization description, overwrites "
+      "options input on command line.");
+  opt("digi-geometric-3d", value<bool>()->default_value(true),
+      "Geometric: Switching geometric digitisation in 3D on");
+  opt("digi-smearing", value<bool>()->default_value(false),
+      "Smearing: Switching geometric digitisation in 3D on");
+  opt("digi-smear-output", value<string>()->default_value(""),
+      "Smearing Output: Name of the output measurement collection.");
+  opt("digi-smear-volume-id",
+      value<read_series>()->multitoken()->default_value({}),
+      "Smearing Input: sensitive volume identifiers.");
+  opt("digi-smear-dimensions",
+      value<read_series>()->multitoken()->default_value({}),
+      "Smearing Input: smear parameters for this volume.");
+  opt("digi-smear-binvalues",
```

Comment:
The name `binValues` might be a little confusing since there is also some binning handled in the uniform smearing. My understanding is that this corresponds to the bound parameter index, right? Could we change it to something like `parIndices`?

Reply:
Good call!

Reply:
Fixed.